source/changelog.txt

```diff
@@ -12,6 +12,14 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v2.0.0
+------
+
+*Upcoming*
+
+- :ref:`Free Monitoring <free-monitoring-mongodb>` is no longer
+  supported for MongoDB Community Edition.  
```

Comment:
This might make it sound a bit as if it has become an enterprise-only feature, or as if Free Monitoring was a feature of mongosh itself; in reality, we just removed helpers in accordance with the deprecation of the feature across all products.

```suggestion
- :ref:`Free Monitoring <free-monitoring-mongodb>` is no longer
  supported through shell helpers.  
```

or something similar?